src/main/java/seedu/address/model/person/Attendance.java

```diff
@@ -53,9 +55,55 @@ public static boolean isValidAttendance(String test) {
         return test.matches(VALIDATION_REGEX);
     }
 
+    /**
+     * Getter method for isPresent
+     * @return boolean value of isPresent
+     */
+    public boolean getIsPresent() {
+        return isPresent;
+    }
+
+    /**
+     * Setter method for isPresent
+     * @param value value is set to isPresent
+     */
+    public void setIsPresent(boolean value) {
+        isPresent = value;
+    }
+
+    /**
+     * Method that marks attendance date as present
+     */
+    public void markAsPresent() {
+        isPresent = true;
+    }
+
+    /**
+     * Method that marks the attendance as absent
+     */
+    public void unmarkAsPresent() {
```

Comment:
Hihi not the best person to review stuff but, do you think its better to name this as markAsAbsent?

or name `markIsPresent` for `markAsPresent`
then name `markIsNotPresent` for `unmarkAsPresent`

since its a boolean toggle iirc they always use isBoolean like that

Reply:
okie, done the necessary changes!! 👍